playbook/spec/pb_kits/playbook/kits/text_input_spec.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require_relative "../../../../app/pb_kits/playbook/pb_text_input/add_on"
```

Comment:
@jasoncypret - This will serve as documentation when we use nested kits as seen in this PR. Do you think any more documentation should be added?